src/main/resources/scripts/-examples/events.sk

```diff
@@ -0,0 +1,44 @@
+
+#
+# This example listens for players joining and leaving.
+# Alters the default message when they do.
+#
+
+on join:
+    set the join message to "Oh look, %player% joined! :)"
+
+on quit:
+    set the quit message to "Oh no, %player% left! :("
+
+#
+# This example cancels damage for players if they have a specific permission.
+# If they don't, tell them how much damage they took.
+#
+
+on damage:
+    victim is a player
+    if the victim has permission "skript.example.damage":
+        cancel the event # Stops the default behaviour - the victim taking damage.
+    else:
+        send "Ouch! You took %damage% damage." to the victim
+        add damage to {damage::%uuid of victim%::taken}
+        if the attacker is a player:
+            add damage to {damage::%uuid of victim%::dealt}
```

Comment:
I assume this is what you want? You're tracking dealt damage so it should be tracking by the attacker's UUID.

```suggestion
            add damage to {damage::%uuid of attacker%::dealt}
```